tests/cli/commands/remote_commands/test_dag_command.py

```diff
@@ -76,9 +81,18 @@ def teardown_class(cls) -> None:
         clear_db_runs()
         clear_db_dags()
 
-    def setup_method(self):
+    def setup_method(self, method):
+        print("running setup methods")
         clear_db_runs()  # clean-up all dag run before start each test
 
+        if method.__name__ == "test_cli_list_dags_prints_import_errors":
+            clear_db_import_errors()
+
+    def teardown_method(self, method):
+        # Clean-up all import errors after the test
+        if method.__name__ == "test_cli_list_dags_prints_import_errors":
+            clear_db_import_errors()
```

Comment:
```suggestion
    def setup_method(self):
        clear_db_runs()
        clear_db_import_errors()

    def teardown_method(self):
            clear_db_import_errors()
```

Just keep it simple and clear for everything here. Less likely for copy/paste down the road to miss the cleanup.